ext/express-parser/antlr4.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/runtime/Cpp/runtime/src/atn/ATNDeserializationOptions.cpp b/runtime/Cpp/runtime/src/atn/ATNDeserializationOptions.cpp
+index a406c4e13..73397107d 100755
+--- a/runtime/Cpp/runtime/src/atn/ATNDeserializationOptions.cpp
++++ b/runtime/Cpp/runtime/src/atn/ATNDeserializationOptions.cpp
```

Comment:
Can we fork the code generator repository so we use it to generate code that doesn’t need patching?

Reply:
Antlr project already fixed this issue.  I updated the version we use from 4.8 to 4.9
Note: Could not upgrade to the latest antlr 4.10 since it is major update with API change